modules/hal_nordic/nrfs/backends/Kconfig

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+menu "NRFS backend settings"
+depends on NRFS
+module = NRFS_BACKEND
+module-str = NRFS backend
+source "subsys/logging/Kconfig.template.log_config"
+
+choice NRFS_IPC_SERVICE_BACKEND_VARIANT
+	prompt "Select variant of IPC service backend for nrfs."
+	depends on NRFS_BACKEND_IPC_SERVICE
+	default NRFS_BACKEND_IPC_SERVICE_LITE
+
+config NRFS_BACKEND_IPC_SERVICE_EXTENDED
+	bool "Zephyr IPC service backend for NRFS with additional buffering."
+
+config NRFS_BACKEND_IPC_SERVICE_LITE
+	bool "Use IPC service directly by NRFS"
```

Comment:
Why two backends and what are the differences?

Reply:
I checked that "standard" version doesn't make sense anymore. I'll remove it

Reply:
removed